edit/edit.js

```diff
@@ -508,7 +517,7 @@ window.onbeforeunload = () => {
   if (isCleanGlobal()) {
     return;
   }
-  updateLintReport(null, 0);
+  updateLintReportIfEnabled(null, 0);
   return confirm(t('styleChangesNotSaved'));
```

Comment:
Hmm, I think the confirmation dialog is actually built-into the browser... if I type in `window.history.back();` in the dev tools console, I see the same confirmation popup.